### PR TITLE
Add bandwidth limit

### PR DESCRIPTION
**Pull request by EmosewaMC**
I havent tested this yet but @DarwinAnim8or said just uncomment it so yeah ill test in morning unless someone else does

### CODE REVIEW

**Comment by Wincent01** [comment]
Maybe make this configurable and allow for it to be turned off.

**Comment by EmosewaMC** [comment]
> Maybe make this configurable and allow for it to be turned off.

Do you know what issues it causes having this limit be higher than it was in live, if it caused any issues to begin with?

File: dNet/dServer.cpp [hunk 1]
```diff
@@ -191,6 +193,11 @@ bool dServer::Startup() {
 	return true;
 }
 
+void dServer::UpdateBandwidthLimit() {
+	auto newBandwidth = mConfig->GetValue("maximum_outgoing_bandwidth");
+	mPeer->SetPerConnectionOutgoingBandwidthLimit(!newBandwidth.empty() ? std::stoi(newBandwidth) : 0);
```
Comment (aronwk-aaron) [inline dNet/dServer.cpp:198]: Does 0 mean unlimited?

**Review by aronwk-aaron — DISMISSED** [review]

File: dNet/dServer.cpp (hunk 1 above)
Comment (EmosewaMC) [reply dNet/dServer.cpp:198]: yes - 0 is unlimited by default as per RakNet documentation

Comment (aronwk-aaron) [reply dNet/dServer.cpp:198]: cool beans

**Review by aronwk-aaron — DISMISSED** [review]

**Review by DarwinAnim8or — COMMENTED** [review]
Code looks fine, but the limit was actually 10kbps, not 100 like it was set in the commented out code. 

I had increased it tenfold to deal with the same Replica stuff I had mentioned before, but that didn't work so I just disabled it all together.

**Comment by EmosewaMC** [comment]
> Code looks fine, but the limit was actually 10kbps, not 100 like it was set in the commented out code.
> 
> I had increased it tenfold to deal with the same Replica stuff I had mentioned before, but that didn't work so I just disabled it all together.

Interesting!  I'll make that change

**Review by DarwinAnim8or — APPROVED** [review]

**Review by aronwk-aaron — APPROVED** [review]

**Pull request merged**